lib/util/reduce_market_data.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+const { PAPER_MODE_PAIRS } = require("../constants");
+
+const INITIAL_MARKETS_OBJECT = {
+  sandboxMarkets: [],
+  liveMarkets: [],
+};
+
+module.exports = (marketData) => {
+  const marketValues = [...marketData.values()];
+
+  return marketValues.reduce((acc, m) => {
+    if (PAPER_MODE_PAIRS.includes(m.wsID)) {
+      acc.sandboxMarkets = [...acc.sandboxMarkets, m];
```

Comment:
there is a bug here, acc is the same ref as the global var `INITIAL_MARKETS_OBJECT`, so on every reload the global state duplicates its size.